CMakeLists.txt

```diff
@@ -62,17 +61,18 @@ endif()
 include(compiler)
 include(os)
 
-if (64_BIT)
+if(64_BIT)
 	set(ARCH "x64")
 else()
 	set(ARCH "x86")
 endif()
 
 message(STATUS "##################################################")
-message(STATUS "Mumble version:  ${version}")
-message(STATUS "Architecture:    ${ARCH}")
+message(STATUS "Mumble version (string): ${VERSION_STRING}")
+message(STATUS "Mumble version:          ${PROJECT_VERSION}")
+message(STATUS "Architecture:            ${ARCH}")
 if(NOT IS_MULTI_CONFIG)
-    message(STATUS "Build type:      ${CMAKE_BUILD_TYPE}")
+    message(STATUS "Build type:              ${CMAKE_BUILD_TYPE}")
```

Comment:
What is this change about?

Reply:
The message was misaligned in the CMake output.

Reply:
Okay. Ideally that should be a separate commit, but given it is such a minor change I guess it's fine...